cmd/openshift-install/agent/waitfor.go

```diff
@@ -106,8 +111,13 @@ func newWaitForInstallCompleteCmd() *cobra.Command {
 				logrus.Fatal(err)
 			}
 
+			authToken, err := agentpkg.FindAuthTokenFromAssetStore(assetDir)
```

Comment:
Given the duplication, maybe the token retrieval step could be pushed down in the `NewCluster` method